src/videojs-contrib-hls.js

```diff
@@ -272,7 +272,8 @@ class HlsHandler extends Component {
     // backwards-compatibility
     if (tech.options_ && tech.options_.playerId) {
       let _player = videojs(tech.options_.playerId);
-
+      videojs.plugin('gapSkipper', gapSkipper);
+      _player.gapSkipper({tech:tech});
```

Comment:
gapskipper should be an instance to player so you can probably just do `this.tech_` inside of it